content/kapp-controller/docs/latest/air-gapped-workflow.md

```diff
@@ -0,0 +1,106 @@
+more concisely in order to focus on applying these workflows to kapp-controller PackageRepositories. 
+
+## Scenario
+
+You have a [PackageRepository](packaging#packagerepository-cr) bundle or image in an external OCI registry that you would like 
+to move into an OCI registry in an air-gapped environment. Once relocated, you would like to deploy the bundle or image as part 
+of a PackageRepository to a Kubernetes cluster.
+
+## Prerequisites
+
+In order to go through this process of moving an [imgpkg bundle](/imgpkg/docs/latest/resources/#bundle) or image to an 
+air-gapped environment, you will need to have [imgpkg](/imgpkg) installed. More information on installing Carvel tools, 
+including `imgpkg` can be found [here](/#whole-suite).
+
+## PackageRepository Format
+
+When going through this air-gapped scenario for kapp-controller PackageRepositories, you will most commonly be relocating a bundle of bundles 
+(i.e. a recursive bundle) from an external OCI registry to an OCI registry in your air-gapped environment. 
+
+When working with kapp-controller PackageRepositories, it is recommended to use the [PackageRepository bundle format](packaging#package-repository-bundle-format). 
+In this format, a PackageRepository is represented as an imgpkg bundle. It is also recommended that Packages that are part of a PackageRepository be imgpkg bundles 
+as documented [here](packaging#package-bundle-format). 
+
+## Copy PackageRepository bundle or image to new location
+
+Most of the steps documented for the [imgpkg air-gapped workflow docs](/imgpkg/docs/latest/air-gapped-workflow#step-1-finding-bundle-in-source-registry) 
+still apply in the case of working with kapp-controller PackageRepositories. A summary of these docs is that you will need to copy your PackageRepository 
+bundle or image with `imgpkg` via one of the following options:
+
+- Option 1: From a common location connected to both registries. This option is more efficient because only changed image layers will be transfered between registries.
+- Option 2: With intermediate tarball. This option works best when registries have no common network access.
+
+More detailed documents for [`Option 1`](/imgpkg/docs/latest/air-gapped-workflow/#option-1-from-a-location-connected-to-both-registries) and 
+[`Option 2`](/imgpkg/docs/latest/air-gapped-workflow/#option-2-with-intermediate-tarball) can be found at the attached links. 
+
+A summary of steps for relocating a PackageRepository bundle or image to an air-gapped environment are documented for both options below:
+
+For `Option 1`: 
+* Get to a location that can access both registries. If there is no such location, you will have to use `Option 2` steps.
+* [Authenticate](/imgpkg/docs/latest/auth.md) with both source and destination registries
+* Run `imgpkg copy -b index.docker.io/user1/simple-app-bundle:v1.0.0 --to-repo registry.corp.com/apps/simple-app-bundle` (Replace `-b` with `-i` for images)
```

Comment:
this should be using `--to-repo final-registry.corp.xyz/apps/simple-app-bundle` since we are tranfering to destination location.